docs/_archive/bugs/bug-306-required-e2e-clerk-session-loss-and-accumulation.md

```
@@ -130,30 +130,43 @@ P3 is therefore proportionate.
 
 ## Resolution State
 
-Open. The red-first lifecycle and diagnostic changes are promoted to `main`
-through PR #835, merge commit `47b31234`. CodeRabbit approved exact head
+Resolved with the original intermittent cause explicitly unproven. The
+red-first lifecycle and diagnostic changes were promoted to `main` through PR
+#835, merge commit `47b31234`. CodeRabbit approved exact head
```

markdownlint-cli2 (0.23.2)

[warning] No space after hash on atx style heading

(MD018, no-missing-space-atx)